src/rgw/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+# local fmt library:
+include_directories(
+  ${PROJECT_SOURCE_DIR}/src/fmt)
+
```

Comment:
I'll look into this (and, after your excellent advice, I was able to get rid of most of these).. but, last I checked, this one was needed. I'll try to see what happens with and without this.

Reply:
i'd guess the problem target is `rgw_admin_user`, which depends on the `rgw_common` object library instead of `rgw_a`. while object libraries can't carry link dependencies, they can carry include directories:
```cmake
target_include_directories(rgw_common PUBLIC ${PROJECT_SOURCE_DIR}/src/fmt)
```

Reply:
I'll get the next set of changes in hopefully tomorrow morning and then try to get my head around this a little better-- I tried fiddling with it and stuff stopped building. ;-) I'll tackle it when the rest of the dust clears.